contracts/fungible-tokens/andromeda-cw20-exchange/src/contract.rs

```diff
@@ -206,8 +206,9 @@ pub fn execute_start_sale(
     );
 
     let end_expiration = if let Some(duration) = duration {
-        // If there's no start time, consider it as now
-        expiration_from_milliseconds(start_time.unwrap_or(current_time) + duration)?
+        // If there's no start time, consider it as now + 1
+        ensure!(duration > 0, ContractError::InvalidExpiration {});
```

Comment:
Why does this need to be 1 second in the future?

Reply:
This returns an error in unit tests without the `+ 1`. I also tried using `.ge` instead, but still returned the same error.
```rust
    ensure!(
        start_expiration.gt(&block_time),
        ContractError::StartTimeInThePast {
            current_time,
            current_block: ctx.env.block.height,
        }
    );
```
Same situation in the crowdfund contract.